src/pages/components/tabs.mdx

```diff
@@ -14,13 +14,23 @@ components:
 <Usage>
   <Use>
     <ul>
-      <li>To provide pre-filtered views of a single dataset</li>
-      <li>To organize related content and controls within a page</li>
+      <li>To filter information into easily parsable views</li>
+      <li>To organize related content and controls within a single page</li>
     </ul>
   </Use>
   <Misuse>
     <ul>
-      <li>As a secondary navigation bar that spans multiple pages</li>
+      <li>
+        To guide users through a task list, use a{' '}
+        <a href="https://zendeskgarden.github.io/react-components/accordions/#stepper">Stepper</a>{' '}
+        instead
+      </li>{' '}
+      <li>
+        {' '}
+        To help users navigate to markedly different kinds of content, use <a href="/components/anchor">
+          Anchors
+        </a> and link to a separate page{' '}
```

Comment:
These leading and trailing spaces shouldn't be needed.

Reply:
Prettier is drunk

Reply:
I quite agree